README.md

```diff
@@ -86,6 +86,9 @@ availability) at this moment.
 go get -u github.com/google/webpackager/cmd/...
 ```
 
+**Note:** Currently Web Packager compiles on Go 1.11+ but you need to enable
+    Go modules (by settting `GO111MODULE=on`) if using Go 1.11.
```

Comment:
1.11 -> 1.11+

Reply:
Fixed. Go 1.13+ doesn't require `GO111MODULE=on`:
https://github.com/golang/go/issues/31857

(I assumed this behavior to be for Go 1.12+ though).